lib/src/main/java/graphql/nadel/NadelSchemas.kt

```diff
@@ -0,0 +1,162 @@
+package graphql.nadel
+
+import graphql.nadel.schema.NeverWiringFactory
+import graphql.nadel.schema.OverallSchemaGenerator
+import graphql.nadel.schema.SchemaTransformationHook
+import graphql.nadel.schema.UnderlyingSchemaGenerator
+import graphql.nadel.util.SchemaUtil
+import graphql.schema.GraphQLSchema
+import graphql.schema.idl.SchemaParser
+import graphql.schema.idl.WiringFactory
+import java.io.Reader
+import java.io.StringReader
+import graphql.schema.idl.ScalarInfo.GRAPHQL_SPECIFICATION_SCALARS as graphQLSpecScalars
+
+data class NadelSchemas internal constructor(
+    val engineSchema: GraphQLSchema,
+    val services: List<Service>,
+) {
+    companion object {
+        @JvmStatic
+        fun newBuilder() = Builder()
+    }
+
+    class Builder {
```

Comment:
People can now use this to create the overall/engine `GraphQLSchema` and the `Service`s.

This separates it from the `Nadel.Builder` so that validation can create this without creating a full Nadel object. The Nadel object is now tied to `NextgenEngine` and also blueprint creation which expects a valid schema. Hence the need to separate them.